### PR TITLE
<Breadcrumbs /> - show ellipsis for long texts

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5843/

File: src/Text/Text.js [hunk 1]
```diff
@@ -3,15 +3,19 @@ import RawText from './RawText';
...
-        weight: props.weight,
-      })}
+      wrapperClassName={st(
+        classes.root,
+        {
+          size: props.size,
+          weight: props.weight,
+        },
+        wrapperClassName,
+      )}
```
Comment (EgoziE) [inline src/Text/Text.js:18]: Adding an external `wrapperClassName` is a good change, please add this exact same change to:
* Caption
* Heading

**Review by EgoziE — CHANGES_REQUESTED** [review]
Please add a story with ellipsis

File: src/Breadcrumbs/docs/index.story.js [hunk 2]
```diff
@@ -103,6 +104,14 @@ export default {
             source: BreadcrumbsThemes,
           }),
 
+          example({
+            title: 'Ellipsis',
+            text: `Breadcrumbs component shows ellipsis when displaying long texts.
+              Hidden text appears on hover in a tooltip.
+              You can control the item's maximum width with the \`itemMaxWidth\` prop, where \`"max-content"\` disables ellipsizing completely.`,
```
Comment (EgoziE) [inline src/Breadcrumbs/docs/index.story.js:111]: 😁 
```suggestion
              You can control the item's maximum width with the \`itemMaxWidth\` prop, where \`"max-content"\` disables ellipsis completely.`,
```

**Review by EgoziE — APPROVED** [review]

**Pull request merged**